packages/lab/src/static-list/StaticListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+import staticListItemCss from "./StaticListItem.css";
+
+const withBaseName = makePrefixer("saltStaticListItem");
+
+export interface StaticListItemProps extends ComponentPropsWithoutRef<"li"> {
+  /**
+   *  To pass a divider to separate the List items. Defaults to false.
+   */
+  divider?: boolean;
```

Comment:
What's the difference between StackLayout's separator with this? 
https://www.saltdesignsystem.com/salt/components/stack-layout/examples#layout-separators

Reply:
used StackLayout as a composition and removed divider